src/test/scala/is/hail/methods/AggregatorSuite.scala

```diff
@@ -390,4 +390,81 @@ class AggregatorSuite extends SparkSuite {
     } (Parameters(rng, 1000, 100))
   }
 
+  private def runAggregatorExpression(expr: String, aggregableName: String, aggregableElementType: Type, aggregableValue: TraversableOnce[_]): Any = {
```

Comment:
Oh nice, I didn't realize `HailContext.eval` existed. Yeah, if we see another use-case for this kind of stuff, we should generalize further.